san-francisco-tourism-solution/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Sun May 13 20:51:52 IST 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
can you add an empty newline at the end of the file (_notice the red crossed out circle_)?

Reply:
I will, but please not that this is an auto-generated file.